CodeEditModules/Modules/WorkspaceClient/src/UI/FileIcon.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  FileIcon.swift
+//  
+//
+//  Created by Nanashi Li on 2022/05/06.
+//
+
+import SwiftUI
+
+// swiftlint:disable function_body_length
+// swiftlint:disable cyclomatic_complexity
+public struct FileIcon {
+    /// Returns a string describing a SFSymbol for files
+    ///
```

Comment:
nit: there is an empty comment

Reply:
To be fair that is for visual separation in the `doc` block

Reply:
got it!